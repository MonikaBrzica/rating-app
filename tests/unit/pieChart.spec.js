import { shallowMount, createLocalVue } from '@vue/test-utils'
import PieChart from '../../src/components/pieChart'
import Vuex from 'vuex'

const localVue = createLocalVue()
localVue.use(Vuex)
const store = new Vuex.Store({
  actions: {
  },
  state: {
  },
  getters: {
  }
})

describe('pieChart', () => {
  const wrapper = shallowMount(PieChart, {
    store,
    localVue
  })
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
