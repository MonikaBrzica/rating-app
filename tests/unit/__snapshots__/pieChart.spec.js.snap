// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pieChart renders correctly 1`] = `
<div
  class="pie-chart"
  id="chart"
>
  <div
    class="title-container"
  >
    <p
      class="title"
    >
      Ratings
    </p>
     
    <img
      alt="..."
      src="../assets/images/dots-vertical.png"
    />
  </div>
   
  <apexchart-stub
    height="380"
    options="[object Object]"
    series=""
    type="pie"
    width="90%"
  />
</div>
`;
